features/dd-sdk-android-trace/src/main/java/com/datadog/opentracing/propagation/HttpCodec.java

```diff
@@ -138,14 +138,35 @@ static BigInteger validateUInt64BitsID(final String value, final int radix)
       throws IllegalArgumentException {
     final BigInteger parsedValue = new StringCachingBigInteger(value, radix);
     if (parsedValue.compareTo(DDTracer.TRACE_ID_MIN) < 0
-        || parsedValue.compareTo(DDTracer.TRACE_ID_MAX) > 0) {
+        || parsedValue.compareTo(DDTracer.SPAN_ID_MAX) > 0) {
```

Comment:
why this change is needed?

Reply:
check the method name...this validates a 64bits number and we introduced another one below for 128 bits. We also introduced a SPAN_ID_MAX and TRACE_ID_MAX (one for 64 and the other for 128). Maybe I should re - name these TRACE_ID_64 and TRACE_ID_128 ?

Reply:
naming should be changed indeed, because using `trace` word for the lower boundary and `span` word for the upper boundary is kind of confusing.